test/wasmfs/wasmfs_getdents.c

```diff
@@ -54,21 +52,23 @@ void print(const char* dir) {
 }
 
 int main() {
-  int err = wasmfs_create_directory("/root", 0777, get_backend());
+  int err;
 
   // Set up test directories.
-  err = mkdir("/root/working", 0777);
+  err = mkdir("root", 0777);
+  assert(err != -1);
+  err = mkdir("root/working", 0777);
   assert(err != -1);
-  err = mkdir("/root/working/test", 0777);
```

Comment:
Why change these files paths to be relative?

Reply:
See https://github.com/emscripten-core/emscripten/blob/4c91962ea50831ba0da77c751017866252e50916/test/setup_nodefs.js#L1-L6